### DISCUSSION TITLE
理性乐观：如何以正增长秩序来看待开源世界 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#3] [Announcements]
# 理性乐观：如何以正增长秩序来看待开源世界 - 开源之道

https://opensourceway.community/posts/open-source-economic/thinking-open-sourc-in-evolve-way/

**Comment by lijiangsheng1** [comment]
😄

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

其实, 这些断言都很飘, 要更加有力量和直接的嗯哼,
什么是货币? 社会必要劳动时间,

只是, 在美元脱离黄金后, 现实社会的货币已经丧失了部分劳动时间的代表;

俺更加期望, FLOSS/自由开源软件世界, 有自己的经济体系, 
不再受世俗经济作品的操弄,

就象多数普通人的普通梦想一样: 只想好好工作, 有尊严的活着,
普通开源社区成员也是: 只想好好写代码, 写出好代码, 同时能有尊严的活着...
